apps/meteor/client/lib/rooms/roomTypes/private.ts

```diff
@@ -88,6 +88,9 @@ roomCoordinator.add(PrivateRoomType, {
 	},
 
 	findRoom(identifier) {
+		if (!hasPermission('view-p-room')) {
+			return;
+		}
```

Comment:
```suggestion
```

It's better not to have this test at all. It'll only cause the room data to be loaded from the server every time.